ci/infra/testrunner/tests/conftest.py

```diff
@@ -47,9 +47,9 @@ def bootstrap(request, provision, skuba):
 @pytest.fixture
 def deployment(request, bootstrap, skuba, kubectl):
     if request.config.getoption("skip_setup") != 'deployed':
-        skuba.join_nodes()
+        skuba.join_nodes(masters=3, workers=2)
```

Comment:
We shouldn't be fixing the number here, as this fixture is used by other tests. It assumes there are some specific number of masters and workers allocated. We must avoid this.